### PR TITLE
Proj module

### PR DESCRIPTION
**Pull request by vdelacruzb**

### CODE REVIEW

**Review by ibrahimmenem — APPROVED** [review]
:+1: for the owned file ci.yml

File: proj/bq/test/proj_integration.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        let rows;
+        await assert.doesNotReject( async () => {
+            const [job] = await client.createQueryJob({ query: query });
+            [rows] = await job.getQueryResults();
+        });
+        assert.equal(rows.length, 1);
+        assert.equal(rows[0].versioncol, 1);
+    });
+
+    it ('Proj method should work', async () => {
```
Comment (Algunenano) [inline proj/bq/test/proj_integration.js:32]: I'm missing some tests with invalid data (invalid projections, invalid/incomplete coordinates) and checks to confirm this works with arrays > 1.

File: proj/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Proj
+
+Based on http://proj4js.org/.
```
Comment (Algunenano) [inline proj/README.md:3]: Please document where the `proj.js` file comes from (source, version..., etc).

**Review by Algunenano — CHANGES_REQUESTED** [review]

**Comment by Jesus89** [comment]
Won't be included for now

**Pull request closed without merging**